### PR TITLE
[bug 1048462] Nix django-compressor

### PR DESCRIPTION
**Pull request by willkg**
We weren't using django-compressor, except for a static finder which
turns out to be mostly a no-op. So we snuck out in the still of the
night and stole the secret plans for the one that kitsune has. NO ONE
WILL DENY US OUR STATIC FILES!

r?

### CODE REVIEW

File: fjord/base/static_utils.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.contrib.staticfiles.finders import BaseStorageFinder
+from django.contrib.staticfiles.storage import StaticFilesStorage
+
+
+class WTFinder(BaseStorageFinder):
```
Comment (rlr) [inline fjord/base/static_utils.py]: HA!!!

**Comment by rlr** [comment]
I nuked the top stati/c dir and loaded the site via devserver and stuff works. WFM
r+

File: fjord/base/static_utils.py (hunk 1 above)
Comment (willkg) [inline fjord/base/static_utils.py]: I stole that from kitsune verbatim. I think you wrote it!

**Comment by willkg** [comment]
649b136 [bug 1048462] Nix django-compressor

Thank you!

**Pull request merged**